content/src/content/jcr_root/apps/core/wcm/components/form/options/v2/options/_cq_dialog/.content.xml

```diff
@@ -230,6 +213,50 @@
                             </columns>
                         </items>
                     </properties>
+                    <constraints
```

Comment:
According to our version policies at https://github.com/adobe/aem-core-wcm-components/wiki/versioning-policies, this new tab would require a new version of the form options component. Would it be possible to add the fields to the existing properties tab?